### PR TITLE
Add test to increase coverage.

### PR DESCRIPTION
**Pull request by JSAssassin**
This PR increases the coverage to 100%

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # bedrock-https-agent ChangeLog
 
+## 2.2.0 - 2022-TBD
+
+### Added
+- Add additional tests.
```
Comment (dlongley) [inline CHANGELOG.md]: There's some private API surface that gets added that should be mentioned.

```suggestion
- Add additional tests and expose private test helper functions in API.
```

File: lib/index.js [hunk 2]
```diff
@@ -54,3 +56,8 @@ bedrock.events.on('bedrock.init', () => {
 function _invalidStateError() {
   return new BedrockError('The agent is not ready.', 'InvalidStateError');
 }
+
+// For testing purposes only
+function _resetHttpsAgent() {
```
Comment (dlongley) [inline lib/index.js]: This doesn't reset the https agent, it clears it, setting it to `null`. Perhaps this should take a param that can be set to `null` and renamed to `setHttpsAgent` instead.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JSAssassin) [reply CHANGELOG.md]: Updated 29b8f78

File: lib/index.js (hunk 2 above)
Comment (JSAssassin) [reply lib/index.js]: Updated 29b8f78

File: lib/index.js [hunk 3]
```diff
@@ -54,3 +56,8 @@ bedrock.events.on('bedrock.init', () => {
 function _invalidStateError() {
   return new BedrockError('The agent is not ready.', 'InvalidStateError');
 }
+
+// For testing purposes only
+function setHttpsAgent({httpsAgent}) {
```
Comment (dlongley) [inline lib/index.js]: Sorry if I was unclear before, this should still be a private function, so `_setHttpsAgent`.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: lib/index.js (hunk 3 above)
Comment (JSAssassin) [reply lib/index.js]: Updated b533851

**Review by dlongley — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Pull request merged**